index.html

```diff
@@ -61,6 +61,205 @@ <h3>LET'S CONNECT</h3>
+          </ul>
+          <button class="portfolio-button">See Project</button>
+        </div>
+
+        <div class="works-card">
+          <img
+            class="works-card--img"
+            src="images/works_images/Snapshoot_Portfolio4.png"
+            alt="project pic"
+          />
+          <h1 class="works-card--title">Multi-Post Stories</h1>
+          <ul class="works-card--info">
+            <li class="canopy">CANOPY</li>
+            <li>
+              <img src="images/works_images/Counter.png" alt="dot icon" />
+            </li>
+            <li>Back End Dev</li>
+            <li>
+              <img src="images/works_images/Counter.png" alt="dot icon" />
+            </li>
+            <li>2015</li>
+          </ul>
+          <p class="works-card--detail">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="works-card--languages">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button class="portfolio-button">See Project</button>
+        </div>
+      </section>
+
+      <section class="about-myself-section">
+        <div class="about-myself--info">
+          <h1 class="about-myself--info--title">About Myself</h1>
+          <p class="about-myself--info--desc">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <div class="about-myself--info--social-media">
+            <h3>LET’S CONNECT</h3>
+            <ul class="about-myself--info--social-media--logo">
+              <li>
+                <a href="https://github.com/zdnahom/"
+                  ><img src="images/github.png" alt="github logo"
+                /></a>
+              </li>
+              <li>
+                <a href="https://www.linkedin.com/in/nahomzerihun76/"
+                  ><img src="images/Linkedin icon.png" alt="Linkedin logo"
+                /></a>
+              </li>
+              <li>
+                <a href="https://angel.co/u/nahom-zerihun-demissie"
+                  ><img
+                    src="images/angellist.png"
+                    alt="angellist(wellfound) logo"
+                /></a>
+              </li>
+              <li>
+                <a href="https://twitter.com/Nahomzerihun11"
+                  ><img src="images/twitter.png" alt="twitter logo"
+                /></a>
+              </li>
+              <li>
+                <a href="https://medium.com/@zerihunnahom"
+                  ><img src="images/medium.png" alt="medium logo"
+                /></a>
+              </li>
+            </ul>
+          </div>
+          <button class="portfolio-button">Get my resume</button>
```

Comment:
- [ ] You may need to consider stating the button `type` attribute. [Type attribute values](https://dev.to/clairecodes/why-its-important-to-give-your-html-button-a-type-58k9) help to state the exact functionality of the button to avoid errors. Kindly consider using [`type='button'`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button) to specific the button functionality :+1: .